src/renderer/components/ft-icon-button/ft-icon-button.vue

```diff
@@ -33,12 +37,19 @@
         <ul
           v-if="dropdownOptions.length > 0"
           class="list"
+          role="listbox"
+          aria-expanded="false"
         >
           <li
             v-for="(option, index) in dropdownOptions"
+            :id="removeWhitespace(title + '-' + index)"
             :key="index"
+            role="option"
+            aria-selected="false"
+            tabindex="-1"
             :class="option.type === 'divider' ? 'listItemDivider' : 'listItem'"
-            @click="handleDropdownClick({url: option.value, index: index})"
+            @click="handleDropdownClick({url: option.value, index: index}, $event)"
+            @keydown="handleDropdownClick({url: option.value, index: index}, $event)"
```

Comment:
Should every keyboard event really trigger the selection or should only specific ones do it?